mlflow/R/mlflow/.run-tests.R

```diff
@@ -9,8 +9,8 @@ use_condaenv(mlflow:::mlflow_conda_env_name())
 keras::install_keras(method = "conda", envname = mlflow:::mlflow_conda_env_name(), tensorflow="1.15.2")
 reticulate::conda_install(Sys.getenv("MLFLOW_HOME", "../../../../."), envname = mlflow:::mlflow_conda_env_name(), pip = TRUE)
 reticulate::conda_install("xgboost", envname = mlflow:::mlflow_conda_env_name())
-reticulate::conda_install("h2o", envname = mlflow:::mlflow_conda_env_name())
-reticulate::conda_install("h2o-py", envname = mlflow:::mlflow_conda_env_name())
+# Pin h2o to prevent version-mismatch between python and R
+reticulate::conda_install("h2o==3.30.1.3", envname = mlflow:::mlflow_conda_env_name(), pip = TRUE)
```

Comment:
Thanks for removing the redundant `h2o-py` installation! A follow-up question: do we need to install the standard h2o Python package as in this line? Or could we also remove that